convert2rhel/unit_tests/checks_test.py

```diff
@@ -1366,7 +1366,7 @@ def test_is_loaded_kernel_latest_eus_system_no_connection(pretend_os, monkeypatc
             8,
             "kernel-core",
             "1",
```

Comment:
To test both the deperecated env var and the new env var, you'll want to modify this so you have the option of specifying which environment variable is set.

Right now, unsupported_skip is the value that the hardcoded environment variable is set to.  You'll need to change that so:
* you are specifying a list of environment variables to set to "1".
* if the list is empty, then no environment variables will be set
* in the unittest's code, you'll have to change it to monkeypatch `os.environ` to set all of the environment variables specified in the `parametrize`.

Reply:
Okay, I get what you are saying here. You are saying that I should set up the test to take a list of the available env vars, and then in parametrize, I can set it to whatever env var I am testing, or if the env var is not set at all. Then down in the test where the asserts are, I can use the monkeypatch with the list of env vars to see if one of them is there or not. Then, I should be able to assert to find the message that should be logged.